.github/workflows/publish.yml

```diff
@@ -61,6 +71,7 @@ jobs:
 
   publish-ldoc:
     if: github.ref == 'refs/heads/master'
+    needs: version-check
```

Comment:
Why is it for the master branch?

Reply:
So, the idea is to stop `publish` jobs in case if mismatch is found between the Lua code and the repository version tag values. The `publish` jobs works for `master` , so it is implemented for it

Related [with](https://github.com/tarantool/expirationd/pull/136#discussion_r1129073685)

Reply:
This job publishes docks. Why should we stop even if we have a problems with tags/versions?

Reply:
Hmm, if it turns out that we forgot to raise the version in the code before the release, we will have to make a new commit. I think it's better to stop all publishing jobs

Reply:
What if I don't need to change the version? Just some PR with a feature.

Reply:
> I think it's better to stop all publishing jobs

I don't think it's relevant for documentation for the master branch.

Reply:
Done, deleted